jekyll/assets/js/sidebar.js

```diff
@@ -2,41 +2,36 @@
 // this improves the visual experience while interacting with the docs site
 (function () {
   window.addEventListener('load', function () {
-    var footer = document.querySelector('.footer');
-    var sidebar = document.querySelector('.sidebar');
-    var defaultSectionName = 'getting-started';
-    var mobileSidebar = document.querySelector('.sidebar-mobile-wrapper');
-    var mobileSidebarDefault = mobileSidebar.querySelector('[data-id="' + defaultSectionName + '"]');
-    var mobileSidebarDisplay = mobileSidebar.querySelector('.mobile-sidebar');
-    var urlParams = new URLSearchParams(window.location.search)
-    var currentSection = urlParams.get("section");
-
-    if (currentSection) {
-      localStorage.sidenavActive = currentSection;
-    }
+    const footer = document.querySelector('.footer');
+    const sidebar = document.querySelector('.sidebar');
+    const defaultSectionName = 'getting-started';
+    const mobileSidebar = document.querySelector('.sidebar-mobile-wrapper');
+    const mobileSidebarDefault = mobileSidebar.querySelector('[data-id="' + defaultSectionName + '"]');
```

Comment:
I think we should still be using es5 until we move this file into src-js and have it properly transpiled?

Reply:
Good point!